src/OrgtrailMonitorLambda/test/orgtrail.test.ts

```diff
@@ -0,0 +1,251 @@
+import * as fs from 'fs';
+import { Configuration } from '../../DeploymentEnvironments';
+import { OrgTrailMonitorHandler } from '../OrgTrailMonitorHandler';
+//import { mockClient} from 'aws-sdk-client-mock';
+//import { SNSClient } from '@aws-sdk/client-sns';
```

Comment:
Kan weg?

Reply:
Jaa of in gebruik nemen als de mock package weer werkt met de SDK zie de link een paar regels onder deze regel.